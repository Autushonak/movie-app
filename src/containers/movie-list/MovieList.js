import React from 'react';
import styled from 'styled-components';
import { MovieCard } from '../../components';

const MovieListStyled = styled.div`
  display: grid;
  grid-template-columns: repeat(3, 1fr);
  grid-row-gap: 50px;
  grid-column-gap: 90px;
  padding: 70px 100px;
`;

const MovieList = ({ movies }) => (
  <MovieListStyled>
    {movies.map(({ url, title, year, genre }) => (
      <MovieCard url={url} title={title} year={year} genre={genre} key={title} />
    ))}
  </MovieListStyled>
);

export default MovieList;
