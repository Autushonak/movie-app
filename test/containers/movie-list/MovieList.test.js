import 'jest-styled-components';
import React from 'react';
import { shallow } from 'enzyme';
import { MovieListStyled } from '../../../src/containers/movie-list/MovieList';

describe('MovieListStyled', () => {
  test('should be rendered correctly', () => {
    const component = shallow(<MovieListStyled />);
    expect(component).toMatchSnapshot();
  });
});
