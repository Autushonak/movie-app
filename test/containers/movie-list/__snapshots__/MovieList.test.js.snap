// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieListStyled should be rendered correctly 1`] = `
.c0 {
  display: grid;
  grid-template-columns: repeat(3,1fr);
  grid-row-gap: 50px;
  grid-column-gap: 90px;
  padding: 70px 100px;
}

<div
  className="c0"
/>
`;
